src/modules/my-positions/selectors/my-positions-summary.js

```diff
@@ -6,21 +6,34 @@ import store from 'src/store'
 import { closePosition } from 'modules/my-positions/actions/close-position'
 
 import { ZERO } from 'modules/trade/constants/numbers'
+import { BUY, SELL } from 'modules/trade/constants/types'
 
 // import { augur } from 'services/augurjs'
 import { formatEther, formatShares, formatNumber } from 'utils/format-number'
 
-export const generateOutcomePositionSummary = memoize((adjustedPosition) => {
+export const generateOutcomePositionSummary = memoize((adjustedPosition, openOrders) => {
   if (!adjustedPosition) {
     return null
   }
+  const simplfiedOrders = openOrders.reduce((accum, item) => {
+    accum[item.type].push({
+      unmatchedShares: item.unmatchedShares.fullPrecision,
+    })
+    return accum
+  }, { [BUY]: [], [SELL]: [] })
+
   const outcomePositions = Array.isArray(adjustedPosition) ? adjustedPosition.length : 1
+  const qtySellSharesOpen = accumulate(simplfiedOrders[SELL], 'unmatchedShares')
+  const qtyBuySharesOpen = accumulate(simplfiedOrders[BUY], 'unmatchedShares')
   const qtyShares = accumulate(adjustedPosition, 'numSharesAdjusted')
+
+  const adjustedQtyShares = qtyShares.isPositive() ? qtyShares.minus(qtySellSharesOpen) : qtyShares.plus(qtyBuySharesOpen)
```

Comment:
Two comments here:

1. I think this adjustment should not be necessary (if I understand how the share balance lookups work correctly).  If the share balances are not being adjusted appropriately when orders are created, that's a bug and should be addressed directly, instead of papering over it by doing an adjustment here.
2. Apart from being unnecessary, I don't think this adjustment is being done correctly here.  The only thing that should affect the user's position is open orders that have shares escrowed.  But again, this adjustment shouldn't be necessary at all, unless I'm misunderstanding something.